examples/game_of_life/game_of_life.serviceconfig

```diff
@@ -11,7 +11,8 @@
       "annotations": {
         "ni/service.description": "MeasurementLink example that displays Conway's Game of Life simulation in a graph.",
         "ni/service.collection": "NI.Examples",
-        "ni/service.tags": []
+        "ni/service.tags": [],
+        "ni/service.programminglanguage": "Python"
```

Comment:
Service authors should not have to add this annotation manually.

The high level design document says:

> In order to log the language the measurement was written in, we will need to modify the LV and Python frameworks to provide this information. This will be done by adding the ni/service.programminglanguage annotation to the measurement.
> 
> ...
>
> In the Python framework, this can be done by modifying [_client.py.register_service](https://github.com/ni/measurementlink-python/blob/bf13d262cdd689f46cd05b2ad3414841dc358296/ni_measurementlink_service/discovery/_client.py#L120)  to include the annotation.

Reply:
Ahh that make sense. Thanks